internal/function/language.go

```diff
@@ -1,12 +1,42 @@
 package function
 
 import (
+	"encoding/binary"
 	"fmt"
+	"hash/crc32"
+	"os"
+	"strconv"
 
+	lru "github.com/hashicorp/golang-lru"
 	enry "gopkg.in/src-d/enry.v1"
 	"gopkg.in/src-d/go-mysql-server.v0/sql"
 )
 
+const (
+	languageCacheSizeKey     = "GITBASE_LANGUAGE_CACHE_SIZE"
```

Comment:
should we add that env variable as we are doing with others? check https://github.com/src-d/gitbase/blob/master/cmd/gitbase/command/server.go#L50

Reply:
If we add it there we will have to pass it all the way down until the infinite. Besides, there are other examples of environment variables such as blob max size that are not there but where they're used